src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_UserWithNullLogin_MotOk() {
+        user.setPassword("password");
+        user.setAge(42);
+        assertThrows(ValidationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_UserWithFiveLengthLogin_NotOk() {
+        user.setLogin("Seven");
+        user.setPassword("fivefsda");
+        user.setAge(20);
+        assertThrows(ValidationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_UserWithSixLengthLogin_Ok() {
+        user.setLogin("Eleven");
+        user.setPassword("RandomPassword");
+        user.setAge(23);
+        registrationService.register(user);
+        User actual = storageDao.get("Eleven");
+        User expected = user;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_UserWithNullPassword_NotOk() {
+        assertThrows(ValidationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_UserWithLoginUnderMinLength_NotOk() {
```

Comment:
register_UserWithFiveLengthLogin_NotOk

isn't it the same test?

Reply:
Yes, they are the same, sorry, I corrected the test so that it matches the name and is not similar